mysite/polls/templates/polls/results.html

```diff
@@ -3,8 +3,7 @@ <h1>{{ question.question_text }}</h1>
 <ul>
     {% for choice in question.choice_set.all %}
         <li>
-            {{ choice.choice_text }} --
-            {{ choice.votes }} vote{{ choice.votes|pluralize }}
+            {{ choice.choice_text }} -- {{ choice.get_votes }} vote{{ choice.get_votes|pluralize }}
```

Comment:
This is making two sql queries, maybe you'd want to cache the choices.get_votes call. I suggest using the `@cached_property` django decorator, in which case I'd rename the field to `vote_count` or something like that since it's more like an attribute than a function

Reply:
Awesome. I'll get on that as soon as possible and push the changes here. Should tests be made to specifically target the cache?

Reply:
Also, if you have time, take a look at the implementation of cached_property which is nice